4-deploy-idsvr.sh

```diff
@@ -74,7 +74,8 @@ fi
 helm repo add curity https://curityio.github.io/idsvr-helm 1>/dev/null
 helm repo update 1>/dev/null
 helm uninstall curity 2>/dev/null
-helm install curity curity/idsvr --values=idsvr/helm-values.yaml
+helm template curity ../idsvr-helm/idsvr --values=idsvr/helm-values.yaml > ~/Desktop/output.yaml 
```

Comment:
Will `~/Desktop/output.yaml` work on Windows?

Not a big fan of this script creating files outside of the project's directory. Maybe at least the delete-cluster script should make a cleanup

Reply:
Accidental checkin - fixed